client-sdk/ts-web/rt/src/accounts.ts

```diff
@@ -0,0 +1,9 @@
+export const MODULE_NAME = 'acccounts';
+
+export const CODE_INVALID_ARGUMENT = 1;
+export const CODE_INSUFFICIENT_BALANCE = 2;
+export const CODE_FORBIDDEN = 3;
+
+export const METHOD_TRANSFER = 'accounts.Transfer';
+export const METHOD_NONCE = 'accounts.Nonce';
+export const METHOD_BALANCES = 'accounts.Balances';
```

Comment:
This should probably differentiate between what are callable methods (e.g. that can be invoked via transactions) and what are query methods.

Reply:
yeah, maybe we can add this along with the request/response typed wrappers

Reply:
also added separator comments for now :shrug: